rholang/src/main/scala/BlockRandomSeed.scala

```diff
@@ -0,0 +1,34 @@
+import coop.rchain.crypto.PublicKey
+import coop.rchain.crypto.hash.Blake2b256.hash
+import coop.rchain.crypto.hash.Blake2b512Random
+import coop.rchain.rspace.hashing.Blake2b256Hash
+import scodec.bits.ByteVector
+import scodec.{Codec, TransformSyntax}
+import scodec.codecs.{bytes, uint16, uint8, ulong, utf8, variableSizeBytes}
+
+final case class BlockRandomSeed(
+    shardId: String,
+    blockNumber: Long,
+    sender: PublicKey,
+    validatorPublicKey: PublicKey,
```

Comment:
In the task description this field wasn't mentioned :thinking:

Reply:
`sender` and `validatorPublicKey` should be refering the same thing.

Reply:
@tgrospic told that both parameters are needed: https://github.com/rchain/rchain/issues/3691#issuecomment-1122161496